sql/core/src/main/java/org/apache/spark/sql/execution/datasources/orc/OrcAtomicColumnVector.java

```diff
@@ -111,10 +103,8 @@ public int getInt(int rowId) {
   @Override
   public long getLong(int rowId) {
     int index = getRowIndex(rowId);
-    if (isTimestamp) {
+    if (isTimestamp && timestampData != null) {
```

Comment:
how can it happen that `isTimestamp` is true but `timestampData` is null?

Reply:
This happens when read timestamp_ntz as timestamp_ltz.